unit-tests/src/test/java/org/eclipse/collections/impl/collector/Collectors2Test2Test.java

```diff
@@ -0,0 +1,646 @@
+        IntList actual =
+                this.smallData.stream().collect(Collectors2.collectInt(each -> each, IntLists.mutable::empty));
+        Assert.assertEquals(expected, actual);
+    }
+
+    @Test
+    public void collectIntParallel()
+    {
+        IntList expected =
+                LARGE_INTERVAL.collectInt(each -> each, IntLists.mutable.empty());
+        IntList actual =
+                this.bigData.parallelStream().collect(Collectors2.collectInt(each -> each, IntLists.mutable::empty));
+        Assert.assertEquals(expected, actual);
+    }
+
+    @Test
+    public void collectFloat()
+    {
+        FloatList expected =
+                SMALL_INTERVAL.collectFloat(each -> (float) each, FloatLists.mutable.empty());
+        FloatList actual =
+                this.smallData.stream().collect(Collectors2.collectFloat(each -> (float) each, FloatLists.mutable::empty));
+        Assert.assertEquals(expected, actual);
+    }
+
+    @Test
+    public void collectFloatParallel()
+    {
+        FloatList expected =
+                LARGE_INTERVAL.collectFloat(each -> (float) each, FloatLists.mutable.empty());
+        FloatList actual =
+                this.bigData.parallelStream().collect(Collectors2.collectFloat(each -> (float) each, FloatLists.mutable::empty));
+        Assert.assertEquals(expected, actual);
+    }
+
+    @Test
+    public void collectLong()
+    {
+        LongList expected =
+                SMALL_INTERVAL.collectLong(each -> (long) each, LongLists.mutable.empty());
+        LongList actual =
+                this.smallData.stream().collect(Collectors2.collectLong(each -> (long) each, LongLists.mutable::empty));
+        Assert.assertEquals(expected, actual);
+    }
+
+    @Test
+    public void collectLongParallel()
+    {
+        LongList expected =
+                LARGE_INTERVAL.collectLong(each -> (long) each, LongLists.mutable.empty());
+        LongList actual =
+                this.bigData.parallelStream().collect(Collectors2.collectLong(each -> (long) each, LongLists.mutable::empty));
+        Assert.assertEquals(expected, actual);
+    }
+
+    @Test
+    public void collectDouble()
+    {
+        DoubleList expected =
+                SMALL_INTERVAL.collectDouble(each -> (double) each, DoubleLists.mutable.empty());
+        DoubleList actual =
+                this.smallData.stream().collect(Collectors2.collectDouble(each -> (double) each, DoubleLists.mutable::empty));
+        Assert.assertEquals(expected, actual);
+    }
+
+    @Test
+    public void collectDoubleParallel()
+    {
+        DoubleList expected =
+                LARGE_INTERVAL.collectDouble(each -> (double) each, DoubleLists.mutable.empty());
+        DoubleList actual =
+                this.bigData.parallelStream().collect(Collectors2.collectDouble(each -> (double) each, DoubleLists.mutable::empty));
+        Assert.assertEquals(expected, actual);
+    }
+}
```

Comment:
Seems so, but it failed with a non-Jacoco error. It might be a Travis issue as well, because the build passes locally.